MAUI/Calendar/events.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: Events in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about the Events support in Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Events in .NET MAUI Calendar (SfCalendar)
+
+The Calendar has the events `ViewChanged`, `SelectionChanged`, `Tapped`, `DoubleTapped`, and `LongPressed` for notifying after user interactions in [.NET MAUI Calendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).
+
+## ViewChanged
+
+The [ViewChanged](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_ViewChanged) event is used to notify when the current view of the calendar is changed while the view is swiped to the previous or next view, as well as when the calendar view is switched to another calendar view.
+
+* `Sender`: This contains the `SfCalendar` object.
+
+* `ViewChanged`: The Calendar current view visible dates are available in the [CalendarViewChangedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarViewChangedEventArgs.html) when the calendar visible dates or view is changed.
```

Comment:
* `ViewChanged`: The Calendar current view visible dates are available in the [CalendarViewChangedEventArgs] -- Check this

because ViewChanged is not a argument

Reply:
Changed the content